with-baseweb/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "private": true,
+  "sideEffects": false,
+  "scripts": {
+    "build": "remix build",
+    "dev": "remix dev",
+    "start": "remix-serve build"
+  },
+  "dependencies": {
+    "@remix-run/node": "*",
+    "@remix-run/react": "*",
+    "@remix-run/serve": "*",
+    "baseui": "^12.2.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
```

Comment:
```suggestion
    "isbot": "^3.6.5",
    "react": "^18.2.0",
    "react-dom": "^18.2.0"
```

Reply:
We'll keep React 17 for now (see https://github.com/remix-run/examples/pull/56#issuecomment-1314298186)